Sources/SnapshotTesting/Strategy.swift

```diff
@@ -1,32 +1,55 @@
 import Foundation
 import XCTest
 
+/// A type representing the ability to transform a value into a diffable format (like text or an image) for snapshot testing.
 public struct Strategy<Snapshottable, Format> {
+  /// The path extension applied to references saved to disk.
   public var pathExtension: String?
+
+  /// How the snapshot format is diffed and converted to and from data.
   public let diffable: Diffable<Format>
+
+  /// How a snapshot is transformed to a diffable format.
   public let snapshotToDiffable: (Snapshottable) -> Async<Format>
```

Comment:
could this be renamed to just `snapshot`? it's the action of turn a snapshottable value into a diffable format.

I'm not sure "How a snapshot is transformed" reads well since it's not really the snapshot but rather the thing being snapshot.